cpp/basix/polyset.cpp

```diff
@@ -80,15 +80,15 @@ void tabulate_polyset_line_derivs(xt::xtensor<double, 3>& P, std::size_t n,
     for (std::size_t p = 1; p <= n; ++p)
     {
       const double a = 1.0 - 1.0 / static_cast<double>(p);
-      xt::col(result, p) = (x0 * 2.0 - 1.0) * xt::col(result, p - 1) * (a + 1.0)
-                           + 2 * k * xt::col(result0, p - 1) * (a + 1.0)
-                           - xt::col(result, p - 2) * a;
+      xt::row(result, p) = (x0 * 2.0 - 1.0) * xt::row(result, p - 1) * (a + 1.0)
+                           + 2 * k * xt::row(result0, p - 1) * (a + 1.0)
+                           - xt::row(result, p - 2) * a;
     }
   }
 
   // Normalise
   for (std::size_t p = 0; p <= n; ++p)
-    xt::view(P, xt::all(), xt::all(), p) *= std::sqrt(2 * p + 1);
+    xt::view(P, xt::all(), p, xt::all()) *= std::sqrt(2 * p + 1);
```

Comment:
I'm not big fan of xtensor's broadcasting.
Shall we use plain loops instead? Maybe in a different PR?